.github/workflows/terraform_lint.yaml

```diff
@@ -0,0 +1,28 @@
+name: Terraform CI - Lint
+
+on:
+  [push]
+
+jobs:
+  tf_lint:
+    name: 'Terraform lint (on tf ${{ matrix.terraform_version }})'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        terraform_version: [ '~1.5.0', '~1.6.0', '~1.7.0', 'latest'] # in theory, we go down to 1.3, but I think that's overkill for lint
```

Comment:
starting rc-0.5 this is >= 1.6
https://github.com/Worklytics/psoxy/blob/rc-v0.5.0/.github/workflows/ci-terraform-modules.yaml
So adapt everywhere please